### PR TITLE
align conf and code

### PR DESCRIPTION
**Pull request by patriknw**
* move some 127.0.0.1 to local conf
* align Main.scala
* improve comments in ShoppingCart

### CODE REVIEW

File: docs-source/docs/modules/cqrs/examples/shopping-analytics-service-scala/src/main/scala/sample/shoppinganalytics/Main.scala [hunk 1]
```diff
@@ -13,16 +15,18 @@ object Main {
...
+    Behaviors.setup[Nothing](context => new Guardian(context))
+  }
+}
 
-      AkkaManagement(system).start()
+class Guardian(context: ActorContext[Nothing]) extends AbstractBehavior[Nothing](context) {
+  val system = context.system
 
-      ShoppingCartEventConsumer.init(system)
+  AkkaManagement(system).start()
```
Comment (ennru) [inline docs-source/docs/modules/cqrs/examples/shopping-analytics-service-scala/src/main/scala/sample/shoppinganalytics/Main.scala:26]: While at it, factor out the Akka Management start as in Shopping cart.

**Review by ennru — APPROVED** [review]
LGTM.

**Review by chbatey — APPROVED** [review]
LGTM

File: docs-source/docs/modules/cqrs/examples/shopping-analytics-service-scala/src/main/scala/sample/shoppinganalytics/Main.scala (hunk 1 above)
Comment (patriknw) [reply docs-source/docs/modules/cqrs/examples/shopping-analytics-service-scala/src/main/scala/sample/shoppinganalytics/Main.scala:26]: right, I missed that

**Pull request merged**